website/docs/notes/concepts.md

```diff
@@ -4,69 +4,104 @@ title: Terminology and Concepts
+| VizWiz               | vizwiz               | vqa           | [website](https://vizwiz.org/)                                                                                    |
+| VQA 2.0              | vqa2                 | vqa           | [website](https://visualqa.org/)                                                                                  |
+| VQA CP V2            | vqacp_v2             | vqa           | [paper: Don’t Just Assume; Look and Answer - Overcoming Priors for Visual Question Answering](https://arxiv.org/pdf/1712.00377.pdf)|
+
+The dataset's key is available under the particular dataset's config, ie., for vizwiz's key, you can look in vizwiz's config available [here](https://github.com/facebookresearch/mmf/blob/master/mmf/configs/datasets/vizwiz/defaults.yaml)
+
+```yaml
+dataset_config:
+  vizwiz: # dataset key
+      data_dir: ${env.data_dir}/datasets
+      depth_first: false
+      fast_read: false
+      use_images: false
+      use_features: true
+      zoo_requirements:
+      - vizwiz.v2019
+      ...
+```
 
 ## Models
 
 Reference implementations for state-of-the-art models have been included to act as a base for reproduction of research papers and starting point of new research. MMF has been used in past for following papers:
 
 - [Towards VQA Models That Can Read (LoRRA model)](https://arxiv.org/abs/1904.08920)
 - [VQA 2018 Challenge winner](https://arxiv.org/abs/1807.09956)
-- VizWiz 2018 Challenge winner
-
-Similar to tasks and datasets, each model has been registered with a unique key for easy reference in configuration and command line arguments. Following table shows each model's key name and datasets it can be run on.
-
-|          |          |                                      |
-| -------- | -------- | ------------------------------------ |
-| Model    | Key      | Datasets                             |
-| LoRRA    | lorra    | vqa2, textvqa, vizwiz                |
-| Pythia   | pythia   | textvqa, vizwiz, vqa2, visual_genome |
-| BAN      | ban      | textvqa, vizwiz, vqa2                |
-| BUTD     | butd     | coco                                 |
-| CNN LSTM | cnn_lstm | clevr                                |
-
-:::note
-
-BAN support is preliminary and hasn't been properly fine-tuned yet.
-
-:::
+- [VizWiz 2018 Challenge winner](https://vizwiz.org/wp-content/uploads/2019/06/workshop2018_slides_FAIR_A-STAR.pdf)
+- [VQA 2020 Challenge winner](https://github.com/facebookresearch/mmf/tree/master/projects/movie_mcan)
+
+Similar to datasets, each model has been registered with a unique key for easy reference in configuration and command line arguments. Following table shows each model's key name and datasets it can be run on. For a more complete list of models, please see [here](https://github.com/facebookresearch/mmf/tree/master/mmf/configs/models)
```

Comment:
I think the main concern in keeping this table is that it creates clutter and is hard to read. Maybe move them into a separate section or the models page itself. Same goes for the dataset table as well. This can be overwhelming for the new user and we should keep it as simple as possible explaining the terminologies and concepts in TLDR format.